scripts/osx/testTemplate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+OF_DIR=`pwd`;
+echo $OF_DIR;
+TEST_DIR="$OF_DIR/scripts/osx"
+cd $TEST_DIR
```

Comment:
We could yes. I can try this. I just didn't want the YML to be confusing and the added benefit that the script would work independently of travis and could be run by anyone to do a quick test.